website/docs/jellyfish/api/loan.md

```diff
@@ -0,0 +1,34 @@
+---
+id: loan
+title: Loan API
+sidebar_label: Loan API
+slug: /jellyfish/api/loan
+---
+
+```js
+import {Client} from '@defichain/jellyfish'
+const client = new Client()
+
+// Using client.loan.
+const something = await client.loan.method()
+```
+
+## createLoanScheme
+
+Creates a loan scheme transaction.
+
+```ts title="client.loan.createLoanScheme()"
+interface loan {
+  createLoanScheme (minColRatio: number, interestRate: BigNumber, options: CreateLoanSchemeOptions): Promise<string>
+}
+
+interface CreateLoanSchemeOptions {
+  id: string
+  utxos?: UTXO[]
+}
```

Comment:
I am thinking we should change how we structure these to always use object, easily to manage optionality.

```ts
client.loan.createLoanScheme({
  minColRatio: 0,
  interestRate: '...',
  id: '',
})

interface CreateLoanSchemeOptions {
  id: string
  minColRatio: number
  interestRate: BigNumber
  utxos?: UTXO[]
}
```

Reply:
```
interface CreateLoanScheme {
  id: string
  minColRatio: number
  interestRate: BigNumber
}

interface CreateLoanSchemeRpc extends CreateLoanScheme {
  utxos?: UTXO[]
}
```
unless I understand utxos here wrongly, is not for fee purpose

Reply:
@jingyi2811 please refactor to:

interface loan {
  createLoanScheme(scheme: CreateLoanScheme, utxos?: UTXO[])
}